src/maybe_dangling.rs

```diff
@@ -25,11 +25,133 @@ use ::core::mem::ManuallyDrop as StdMD;
 ///
 /// For the adventurous `nightly` users, you can enable the
 /// `nightly-dropck_eyepatch` Cargo feature to opt into the usage of the
-/// eponymous `rustc` feature so as to get the `Drop` impl amended accordingly.
+/// [eponymous `rustc` feature][RFC-1327] so as to get the `Drop` impl amended
+/// accordingly.
+///
+/// Below are three scenarios that should illustrate how Drop Check (`dropck`)
+/// handles transitive drop glue when checking for dangling pointers.
+/// All three scenarios are theoretically sound, since no dangling pointer is
+/// ever accessed in any destructor.
+/// But none will compile without the `nightly-dropck_eyepatch` feature there
```

Comment:
very itsy tiny nit:

```suggestion
/// But none will compile without the `nightly-dropck_eyepatch` feature enabled
```

(imho this way it reads slightly better)